src/App.tsx

```diff
@@ -5,16 +5,18 @@ import Login from "./components/auth/Login";
 import { useDispatch, useSelector } from "react-redux";
 import { statesModal } from "./store/todoReducer";
 import Button from "./components/shared/form/Button";
-import { loginTodoAction } from "./store";
+import { setTodoAction, loginTodoAction } from "./store";
+import instance from "./axiosConfig";
```

Comment:
You can keep the name as `axios` or change it to `axiosInstance` for readability.

Reply:
updated to axiosInstance